DataFormats/Scouting/test/test_readRun3Scouting_cfg.py

```diff
@@ -9,10 +10,14 @@
 parser.add_argument("--vertexVersion", type=int, help="photon data format version (default: 4)", default=4)
 parser.add_argument("--inputFile", type=str, help="Input file name (default: testRun3Scouting.root)", default="testRun3Scouting.root")
 parser.add_argument("--outputFileName", type=str, help="Output file name (default: testRun3Scouting2.root)", default="testRun3Scouting2.root")
+parser.add_argument("-f", "--fixStreamerInfo", action="store_true")
```

Comment:
what' the purpose of this option? Looks like the `fixReading_12_4_X_Files` function is invoked below at L18 regardless.

Reply:
Yeah, the call to `fixReading_12_4_X_File` should have beeb kept inside`if args.fixStreamerInfo:` (sorry if I was unclear)

Reply:
Oops. Just a mistake on my part. I would suggest merging this PR anyway to keep the unit tests working and then I will follow up with a small separate PR fixing that.

Reply:
If requested, I'll immediately put in a fix. But then we'll have to wait for tests to run and signatures... A separate PR that only affects one file will be easier.